app/src/main/res/values-nl/strings.xml

```diff
@@ -0,0 +1,292 @@
+<resources>
+    <string name="app_name">QR-scanner</string>
+
+    <string name="navigation_drawer_open">Navigatiemenu openen</string>
+    <string name="navigation_drawer_close">Navigatiemenu sluiten</string>
+    <string name="qr_scanner">QR-scanner</string>
+    <string name="qr_generator">QR-generator</string>
+    <string name="history">Geschiedenis</string>
+    <string name="settings">Instellingen</string>
+    <string name="help">Hulp</string>
+    <string name="about">Over</string>
+    <string name="title_activity_qrscanner">QR-scanner</string>
+
+    <string name="action_settings">Instellingen</string>
+    <string name="title_activity_qr_scanner">QR-scanner</string>
+    <string name="title_activity_qr_generator">QR-generator</string>
+    <string name="title_activity_history">Geschiedenis</string>
+    <string name="title_activity_settings">Instellingen</string>
+    <string name="title_activity_help">Hulp</string>
+    <string name="title_activity_about">Over</string>
+    <string name="Settings">Instellingen</string>
+    <string name="QRScanner">QR-scanner</string>
+    <string name="QRGenerator">QR-generator</string>
+    <string name="History">Geschiedenis</string>
+
+    <string name="slide1_heading">Welkom!</string>
+    <string name="slide1_text">Welkom bij de privacyvriendelijke QR-scanner. In deze rondleiding krijg je de belangrijkste functies te zien.</string>
+    <string name="slide2_heading">Wat kan ik doen?</string>
+    <string name="text_slide2">Je kunt een QR-code scannen met de QR-scanner of een QR-code genereren met de QR-generator.</string>
+    <string name="slide3_heading">Meer functies</string>
+    <string name="text_slide3">De gescande QR-code kun je opslaan in de geschiedenis, delen of kopi?ren.</string>
+    <string name="wecome">Welkom</string>
+    <string name="skip">Overslaan</string>
+    <string name="next">Volgende</string>
+    <string name="help_">Hulp</string>
+
+    <string name="tutorial">Rondleiding</string>
+
+    <!--###ABOUT###-->
+    <string name="about_author">Makers: </string>
+    <string name="about_author_names" translatable="false">Christopher Beckmann, Bassel Hanna, Philipp Rack</string>
```

Comment:
Non translatable entries should not be added to a translation.

Reply:
Did not know that. Fixed as of now.